index.html

```diff
@@ -113,15 +129,15 @@ <h2>Multi-Post Stories</h2>
     </div>
   </div>
   <div class="works_card">
-    <img src="./images/tonic.svg" alt="SnapshootPortfolio">
+    <img src="./images/works/tonic.svg" alt="SnapshootPortfolio" class="works_img">
     <div class="works_card_detail">
       <div class="works_card_title">
         <h2>Tonic</h2>
         <div class="works_card_history">
            <div class="works_card_history_detail"><p>CANOPY</p></div>
-           <div><img src="./images/Counter.svg" alt=""></div>
+           <div><img src="./images/works/Counter.svg" alt=""></div>
            <div class="works_card_history_detail"><p>Back End Dev</p></div>
-           <div><img src="./images/Counter.svg" alt=""></div>
+           <div><img src="./images/works/Counter.svg" alt=""></div>
            <div class="works_card_history_detail"><p>2015</p></div>
         </div>
       </div>
```

Comment:
- You have done well so far, but please align this section properly, I suggest you use the `flex` display property to align the content of your project cards because it is responsive. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏. Please fix this for all your project cards.

Your project:
![hehehe](https://github.com/katarighe/my-portfolio/assets/73607512/d53aecc1-75eb-4fcd-81c0-a672c77e2bc6)

Design template:
![erhegh](https://github.com/katarighe/my-portfolio/assets/73607512/1ed787db-2860-4339-b480-2c219dcebaf6)